pkgs/tools/admin/awscli2/default.nix

```diff
@@ -29,13 +24,13 @@ let
 in
 with py.pkgs; buildPythonApplication rec {
   pname = "awscli2";
-  version = "2.1.17"; # N.B: if you change this, change botocore to a matching version too
+  version = "2.1.29"; # N.B: if you change this, change botocore to a matching version too
```

Comment:
Does botocore require an update?

Reply:
Yeah, it's being bumped in this PR:
https://github.com/NixOS/nixpkgs/blob/6f4b61d2ce39057fc0d9971fa7620cc7d7e7dd3a/pkgs/tools/admin/awscli2/default.nix#L6